test/fixtures/wallet-web/.env

```diff
@@ -29,6 +29,10 @@ HUB_ROUTER_API_PORT=10093
 HUB_ROUTER_DB_PATH=/tmp/db/aries1
 HUB_ROUTER_WEBHOOK_PORT=10094
 HUB_ROUTER_AUTOACCEPT=true
+HUB_ROUTER_USE_DIDCOMM_V2=true
+HUB_ROUTER_ORB_DOMAINS=https://testnet.orb.local
+HUB_ROUTER_V2_KEY_TYPE=ecdsap256ieee1363
+HUB_ROUTER_V2_KEY_AGREEMENT_TYPE=p256kw
```

Comment:
Are you sure this config works for V1?

Reply:
@sudeshrshetty it's ok for now. We can fix it when we test DIDcomm v1